docs/user-guide/getting-started.md

```diff
@@ -6,31 +6,30 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 * ESLint uses an AST to evaluate patterns in code.
 * ESLint is completely pluggable, every single rule is a plugin and you can add more at runtime.
 
-## Getting Started Tutorial
-
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/hppJw2REb8g?rel=0" frameborder="0" allowfullscreen></iframe>
-*Why ESLint* @0:00, *Installing and using ESLint* @2:20.  <a href="https://www.pluralsight.com/courses/eslint-better-code-quality?utm_source=eslint-dot-org&utm_medium=video&utm_campaign=authordemo" target="_blank">Full ESLint Course at Pluralsight</a>
-
 ## Installation and Usage
 
 Prerequisites: [Node.js](https://nodejs.org/en/) (`^8.10.0`, `^10.13.0`, or `>=11.10.1`) built with SSL support. (If you are using an official Node.js distribution, SSL is always built in.)
 
-You can install ESLint using npm:
+You can install ESLint using npm or yarn:
 
 ```
-$ npm install eslint --save-dev
+npm install eslint --save-dev
+
+# or
+
+yarn add eslint --dev
```

Comment:
I think this is a good idea! Do we have any specific policies about  our `yarn` support? I ask because listing this here I think signals that we do support it. It's popular enough that I think it makes a lot of sense commit to supporting it.

Reply:
Yarn is just a package manager like npm. I think there's nothing special here.

Reply:
Agree. I don’t think Yarn is something we “support”. It’s something you can use to install ESLint.

Reply:
I personally agree. I bring this up because I seem to remember a few issues over the last couple years where our answer to a Yarn-specific issue was along the lines of, “we officially support `npm`”. That being said, I tried to find some examples and wasn’t successful, so there clearly weren’t many of them.

Reply:
My recollection is that when yarn was less mature, we had some issues around why yarn would fail to install dependencies correctly and things like that, and it became frustrating to try to understand a package manager that seemed like it just didn't do the same thing as npm and didn't always conform to node package conventions. Yarn is now much more mature, so it's not as much of a maintenance burden to "support" Yarn.